cmd/etcd-shield/main.go

```diff
@@ -102,12 +107,14 @@ func main() {
 	var tlsCert string
 	var tlsKey string
 	var configPath string
+	var metricsAddr string
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false, "Enable leader election.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.IntVar(&webhookPort, "port", 9443, "Port to listen for webhook events on.")
 	flag.StringVar(&tlsCert, "tls-cert", "/var/tls/tls.crt", "File location of tls certificate.")
 	flag.StringVar(&tlsKey, "tls-key", "/var/tls/tls.key", "File location of tls key pair.")
 	flag.StringVar(&configPath, "config", "/etc/etcd-shield/config.yaml", "Location of etcd-shield config")
+	flag.StringVar(&metricsAddr, "metrics-addr", "0.0.0.0:9100", "Address to serve metrics on")
```

Comment:
Looking at L112, should we default this to just `:9100`?


```suggestion
	flag.StringVar(&metricsAddr, "metrics-addr", ":9100", "Address to serve metrics on")
```

Reply:
Fixed in de77a05.